pom.xml

```diff
@@ -17,7 +17,7 @@
     <packaging>pom</packaging>
 
     <properties>
-        <quarkus.version>2.9.2.Final</quarkus.version>
+        <quarkus.version>2.10.0.Final</quarkus.version>
```

Comment:
In order to have the OTLP registry, I needed to bump Quarkus... the OTLP registry was added in micrometer `1.9.0` and Quarkus `2.10.0.Final` bumped micrometer to that version